pkg/addons/data/ClusterRole.rbac.authorization.k8s.io/dedicated-cluster-admin.yaml

```diff
@@ -0,0 +1,167 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: dedicated-cluster-admin
+rules:
+- apiGroups:
+  - ''
+  - user.openshift.io
+  resources:
+  - groups
+  - users
+  - identities
+  - useridentitymappings
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - ''
+  - authorization.openshift.io
+  resources:
+  - rolebindings
+  - clusterrolebindings
+  verbs:
+  - create
+  - delete
+  - get
+  - list
+  - patch
+  - update
+  - watch
+- apiGroups:
+  - rbac.authorization.k8s.io
```

Comment:
It seems you can collapse this set of permissions with the above set
```yaml
- apiGroups:
  - ''
  - authorization.openshift.io
  - rbac.authorization.k8s.io
  resources:
  - rolebindings
  - clusterrolebindings

Reply:
will do.